docs/tutorials/how-tos/testing-devnet-features.md

```diff
@@ -0,0 +1,203 @@
+---
+html: testing-devnet-features.html
+parent: how-tos.html
+seo:
+    description: Guide on using pre-release transaction types on the XRP Ledger for developers.
+labels:
+  - Development
+  - Customization
+  - Devnet
+---
+# Test a Pre-Release Transaction Types
+
+_(Requires cloning and modifying XRPL core repositories and understanding of XRPL [transaction serialization](../../references/protocol/binary-format.md))_
+
+Pre-release transactions are [amendments](../../concepts/networks-and-servers/amendments.md) that represent new features or other changes to transaction processing. Features are typically released to the [XRPL Devnet](../../concepts/networks-and-servers/parallel-networks.md) for early testing.
+
+This guide walks through using either JavaScript with `xrpl.js` or Python with `xrpl-py`. This approach is typically only necessary for early pre-release amendments on the [XRPL Devnet](../../concepts/networks-and-servers/parallel-networks.md).
+
+**Note**: The code samples below illustrate how to prepare your development environment and modify the XRPL to support custom transaction types, using the respective [client library](../../references/client-libraries.md).
+
+## Prerequisites
+
+- Basic understanding of XRPL transactions.
+- Development environment setup for JavaScript or Python.
+- Docker installed and configured.
+
+## Steps
+
+### 1. Set Up Your Development Environment
+
+Ensure the proper dependencies are installed for Node.js or Python.
+
+{% tabs %}
+
+{% tab label="JavaScript" %}
+
+```javascript
+npm install xrpl
+```
+
+{% /tab %}
+
+{% tab label="Python" %}
+
+```bash
+pip install xrpl-py
+```
+
+{% /tab %}
+
+{% /tabs %}
+
+### 2. Generate Definitions File
+
+Utilize the [server_definitions](../../references/http-websocket-apis/public-api-methods/server-info-methods/server_definitions.md) command to retrieve the definitions.json content.
+
+**Note:** Any [parallel network](../../concepts/networks-and-servers/parallel-networks.md) may be used instead of Devnet.
```

Comment:
Given the goal of this tutorial is to test amendments in development, should we say "any parallel test network" instead?

Reply:
Up to you, I'm good either way

Reply:
Updated to "parallel test network"